jsonld-contexts/waterway-observed-attributes.jsonld

```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "ObservedWaterFlow": "https://smartdatamodels.org/ObservedWaterFlow",
+    "WeatherCondition": "https://smartdatamodels.org/WeatherCondition",
+    "adminComment": "https://smartdatamodels.org/adminComment",
+    "comment": "https://smartdatamodels.org/comment",
```

Comment:
`comment` attribute is already specified in `shellfish-farming.jsonld`

Reply:
fixed.